Added option fieldSeparator

kameamea opened the pull request.

nathanb commented:
Nice work! I like this too.  Be sure to update the tests; one is failing.  If you're familiar with rebase/squashing, crunch it down to one commit and we're good to go. (If not, no problem).

kameamea commented:
I fixed the test (also I didn't like to duplicate the fieldSeparator initialisation into the constructor, but still seemed the best way to do it). Then I also added two tests checking the fieldSeparator code.

I'm sorry, but I have no experience with rebase/squashing. From what I've read about it I'm not sure whether I should do this on my local repository or on github. And on the latter I don't know how I would be able to do it. Well, I could create a new branch with only one commit if this helps?

nathanb commented:
No worries at all! Nice work! Thanks!  I'll merge and push out a release real quick.
